src/ui/MessageContent/MessageBody/index.tsx

```diff
@@ -84,15 +83,23 @@ export const MessageBody = (props: MessageBodyProps) => {
           form={message.messageForm}
         />
       ))
-    .when(isTemplateMessage, () => (
-      <TemplateMessageItemBody
+    .when(isTemplateMessage, () => {
+      if (!isValidTemplateMessageType(message.extendedMessagePayload[MESSAGE_TEMPLATE_KEY])) {
+        return <UnknownMessageItemBody
+          className={className}
+          message={message}
+          isByMe={isByMe}
+          mouseHover={mouseHover}
+          isReactionEnabled={isReactionEnabledInChannel}
+        />;
+      }
```

Comment:
Unknown 이 렌더링하는곳은 이미 아래에 존재하기때문에, 한곳에서만 렌더링 하도록 깔끔하게 정리가 되면 좋을 것 같습니다.

Reply:
요렇게 안하면 isTextMessage 가 true 가 되서 TextMessageItemBody 로 렌더링 되어서 이렇게 수정했습니다.

Reply:
네, 그걸 고려해서 렌더링 하는 부분을 묶던가 하면 좋을거 같아서 말씀드려보았습니다.

Reply:
좋은 아이디어가 떠오르질 않습니다. 아이디어 있으시면 설명 부탁드릴게요.

Reply:
```tsx
const renderUnknown = () => {
  return <Unknown />
}

match()
  .when(xx, () => {
    if (invalid) return renderUnknown();
    return <MessageTemplate />
  })
  .otherwise(() => {
    return renderUnknown();
  })
```

Reply:
<img width="888" alt="Screenshot 2024-09-25 at 4 46 58 PM" src="https://github.com/user-attachments/assets/65150a11-5fe5-4634-a3b4-7ec6c98c4558">

제가 구현한거랑 위에 예시랑 다른가요..?

Reply:
아 unknown 을 wrapper 로.. 이해했습니다.